package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pcln-design-system",
-  "version": "1.0.0-1",
+  "version": "1.1.0-1",
```

Comment:
Leave the version number as-is until after we merge. The other PRs are likely to go to the next release as well. Also fwiw, the `-1` denotes a beta version and we wanna hold off on a v1 until things are a little more stable

Reply:
Great thought. Thanks for sharing. Reverted the version back to `1.0.0-1`.